src/Apps/Settings/Routes/Insights/InsightsRoute.tsx

```diff
@@ -33,6 +34,8 @@ const InsightsRoute: React.FC<InsightsRouteProps> = ({ me }) => {
 
   return (
     <>
+      <MetaTags title="Insights | Artsy" pathname="/insights" />
```

Comment:
I wonder if `My Collection Insights | Artsy` would be a better title 🤔 

```suggestion
      <MetaTags title="My Collection Insights | Artsy" pathname="/insights" />
```

Reply:
Yes, it's a better title actually :)